src/mammos_entity/io.py

```diff
@@ -62,7 +68,9 @@
 
 
 def entities_to_csv(
-    filename: str | Path,
+    _filename: str | Path,
+    _description: str | None = None,
+    /,
     **entities: mammos_entity.Entity | mammos_units.Quantity | numpy.typing.ArrayLike,
 ) -> None:
     """Write tabular data to csv file.
```

Comment:
We should add and `Args:` section to the docstring and also mention that filename and description must be positional arguments.

I would also mention that the leading underscore is used to still allow an entity-like `description` or `filename` in the data.